packages/lib/response/tests/constants.ts

```diff
@@ -0,0 +1,23 @@
+export function randUuid(): string {
```

Comment:
the function name doesn't really make sense now since we don't have random values. can we maybe also change this to constants?